### PR TITLE
Migrate wfMsg to wfMessage

### PR DESCRIPTION
**Pull request by kghbln**

### CODE REVIEW

File: src/SpecialSemanticSignup.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ private function userSignup() {
 			$status = $user->sendConfirmationMail();
 
 			if ( !$status->isGood() ) {
-				throw new Exception( wfMsg( 'ses-emailfailed' ) . "\n" . $status->getMessage() );
+				throw new Exception( wfMessage( 'ses-emailfailed' )->escaped() . "\n" . $status->getMessage() );
```
Comment (siebrand) [inline src/SpecialSemanticSignup.php]: Why do you escape here and not above? The 1-to-1 conversion of wfMsg is wfMessage()->text().

File: src/UserAccountDataChecker.php [hunk 2]
```diff
@@ -63,26 +63,26 @@ public function checkSorbs() {
...
-			$this->error( wfMsg( 'ses-userexists' ) );
+			$this->error( wfMessage( 'ses-userexists' )->text() );
 	}
 
 	public function checkPasswordLength() {
 		if ( !$this->mUser->isValidPassword( $this->mPassword ) )
 		{
 			global $wgMinimalPasswordLength;
-			$this->error( wfMsgExt( 'passwordtooshort', array( 'parsemag' ), $wgMinimalPasswordLength ) );
+			$this->error( wfMessage( 'passwordtooshort' )->numParams( $wgMinimalPasswordLength )->text() );
```
Comment (siebrand) [inline src/UserAccountDataChecker.php:78]: Nice. You fixed an i18n bug in the process. Now that you're using numParams(), numeral conversion is also applied automagically.

Comment (kghbln) [inline src/UserAccountDataChecker.php:78]: Cool, thanks for the encouragement. :) I guess I really learned something.

File: src/SpecialSemanticSignup.php (hunk 1 above)
Comment (kghbln) [inline src/SpecialSemanticSignup.php]: Cannot really remember. Probably I thought that this is part of a form "email-field" O_o Thanks for the hint!

**Pull request merged**